src/vendor/wordpress/core-blocks.stories.mdx

```diff
@@ -466,10 +466,12 @@ applied only if the solid color background is toggled on.
 
 <Canvas>
   <Story name="Pullquote">
-    {`<figure class="wp-block-pullquote has-background is-style-solid-color" style="background-color:#dcd7ca">
+    {`<figure class="wp-block-pullquote has-background is-style-solid-color">
         <blockquote class="has-text-color has-accent-color">
           <p>One of the hardest things to do in technology is disrupt yourself.</p>
-          <cite>Matt Mullenweg</cite>
+          <footer>
+            <cite>Matt Mullenweg</cite>
+          </footer>
```

Comment:
Did you verify that WordPress includes the `footer` element? If not, then we probably can't modify this markup.

Reply:
I didn't, I'll double check that.